### PR TITLE
Feature vishal/header implimentation

### PR DESCRIPTION
**Pull request by vishal-panorama**
Implemented Header 
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,49 +1,53 @@
...
+import Home from './MainContent/Home';
+import MainHeader from './MainHeader';
+import MainContent from './MainContent';
+
 
 function App() {
+  const [modal, setModal] = useState(false);
+
   return (
     <div >
```
Comment (pkumar7777) [inline src/App.js:28]: div not needed

File: src/Footer/CopyrightPolicy/index.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import React from 'react'
+import { Stack } from 'react-bootstrap'
 
 const CopyrightPolicy = () => {
   return (
     <div>
```
Comment (pkumar7777) [inline src/Footer/CopyrightPolicy/index.js:6]: use fragment( <></>) rather div for enclosing multiple elements

File: src/Footer/index.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
 import Fdata from './Fdata'
-import './Footer.css'
+import CopyrightPolicy from './CopyrightPolicy';
+import './Footer.css';
 const Footer = () => {
     return (
         <div>
```
Comment (pkumar7777) [inline src/Footer/index.js:8]: divs not needed

File: src/MainContent/Cartmodal/index.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                            <span>Sr.</span>
+                            <span>Item Description</span>
+                            <span>Quantity</span>
+                            <span>Item Price</span>
+                            
+                        </div>
+                        </Stack>
+                       
+                    </Modal.Title>
+                    <CloseButton  onClick={initModal}/>
```
Comment (pkumar7777) [inline src/MainContent/Cartmodal/index.js:29]: please use setModal(false) , no need to create a new function initModal

File: src/MainContent/Cartmodal/index.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import './Cartmodal.css';
+import Stack from 'react-bootstrap/Stack';
+import { Modal, Button, CloseButton } from 'react-bootstrap'
+export default function Cartmodal({ setModal, modal }) {
+    const initModal = () => {
```
Comment (pkumar7777) [inline src/MainContent/Cartmodal/index.js:6]: remove this function as per below comment

File: src/MainContent/Cartmodal/index.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+                    Your Shopping Cart is currently empty. To add Books in your Shopping Cart, start by searching or browsing through our book store. When a book interests you, click on Add to Cart button. Books in Shopping Cart always reflect the most recent price, displayed on their product pages.
+               
+                    Your Shopping Cart is currently empty. To add Books in your Shopping Cart, start by searching or browsing through our book store. When a book interests you, click on Add to Cart button. Books in Shopping Cart always reflect the most recent price, displayed on their product pages.
+                    Your Shopping Cart is currently empty. To add Books in your Shopping Cart, start by searching or browsing through our book store. When a book interests you, click on Add to Cart button. Books in Shopping Cart always reflect the most recent price, displayed on their product pages.
+               
+                </Modal.Body>
+                <Modal.Footer>
+                    <Button variant="danger" onClick={initModal}>
```
Comment (pkumar7777) [inline src/MainContent/Cartmodal/index.js:40]: same as above

File: src/MainContent/ContactUs/Contact.css [hunk 7]
```diff
@@ -0,0 +1,86 @@
+body {
```
Comment (pkumar7777) [inline src/MainContent/ContactUs/Contact.css:1]: body css should be in common css, please use a single css for app level changes

File: src/MainHeader/Header/CartIcon/index.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+
+export default function CartIcon({ setModal }) {
+  const handelCart = () => {
```
Comment (pkumar7777) [inline src/MainHeader/Header/CartIcon/index.js:5]: same as above remove this

File: src/MainHeader/Header/Favorite/index.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+const Favorite = () => {
+    const handelAccount =()=>{
```
Comment (pkumar7777) [inline src/MainHeader/Header/Favorite/index.js:3]: empty function remove this

**Review by pkumar7777 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (vishal-panorama) [reply src/App.js:28]: Resolved

File: src/Footer/index.js (hunk 3 above)
Comment (vishal-panorama) [reply src/Footer/index.js:8]: Needed div Because returning multiple Tags (div)

File: src/MainContent/Cartmodal/index.js (hunk 4 above)
Comment (vishal-panorama) [reply src/MainContent/Cartmodal/index.js:29]: Resolved.

File: src/MainContent/Cartmodal/index.js (hunk 6 above)
Comment (vishal-panorama) [reply src/MainContent/Cartmodal/index.js:40]: Done.

File: src/MainContent/ContactUs/Contact.css (hunk 7 above)
Comment (vishal-panorama) [reply src/MainContent/ContactUs/Contact.css:1]: done.

File: src/MainHeader/Header/CartIcon/index.js (hunk 8 above)
Comment (vishal-panorama) [reply src/MainHeader/Header/CartIcon/index.js:5]: done.

File: src/MainHeader/Header/Favorite/index.js (hunk 9 above)
Comment (vishal-panorama) [reply src/MainHeader/Header/Favorite/index.js:3]: Done.

**Pull request merged**